composer.json

```diff
@@ -19,7 +19,8 @@
         "phpunit/dbunit": "3.0.*",
         "facebook/webdriver" : "dev-master",
         "phpmd/phpmd": "^2.6",
-        "phan/phan": ">1.2.2"
+        "phan/phan": "~2.2.5",
+        "microsoft/tolerant-php-parser": "~0.0.18"
```

Comment:
This is a require dependency of phan.

Reply:
then it shouldn't be added here, dependencies are resolved recursively.

Reply:
I see what you're saying. I tried editing `composer.json` and then running `composer update phan/phan` so that other packages didn't update. I got a conflict with the microsoft package so I added this line here.

Is there a better way?

Reply:
I just tried changing the phan version to `>2.2` locally and composer didn't have any conflicts.

```
➜  Loris git:(d17aa5d5d) ✗ composer update phan/phan
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Updating phan/phan (1.3.1 => 2.2.3): Downloading (100%)         
Package phpunit/dbunit is abandoned, you should avoid using it. No replacement was suggested.
Package phpunit/phpunit-mock-objects is abandoned, you should avoid using it. No replacement was suggested.
Writing lock file
Generating autoload files
```

Reply:
It looks like you're installing version 2.2.3 where my update gets us to 2.2.5

Does the same thing happen if you install the newest version that way?

Reply:
works fine so long as I update both dependencies:

```
$ composer update phan/phan microsoft/tolerant-php-parser
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 2 updates, 0 removals
  - Updating microsoft/tolerant-php-parser (v0.0.17 => v0.0.18): Downloading (100%)         
  - Updating phan/phan (2.2.3 => 2.2.5): Downloading (100%)         
Package phpunit/dbunit is abandoned, you should avoid using it. No replacement was suggested.
Package phpunit/phpunit-mock-objects is abandoned, you should avoid using it. No replacement was suggested.
Writing lock file
Generating autoload files
```

Reply:
OK I'll redo this

Reply:
Thanks for reviewing and testing so quickly.